app/reducers/builderReducer.ts

```diff
@@ -223,6 +225,7 @@ export function initialBuilderState({
     userTags,
     undoHistory: [fromCells(initialGrid)],
     undoIndex: 0,
+    cluesBackup: null,
```

Comment:
I ended up putting this on state in order to keep the actions thin, and because an action that sets all clues arbitrarily seemed too powerful somehow...

Reply:
Makes sense!